fishtest/fishtest/templates/base.mak

```diff
@@ -2,94 +2,79 @@
 <html>
 <head>
   <title>Stockfish Testing Framework</title>
-  <link href="/css/bootstrap.min.css" rel="stylesheet">
-  <%block name="meta"/>
 
-  <script src="https://code.jquery.com/jquery-1.8.3.js"></script>
-  <script src="/js/bootstrap.min.js"></script>
-  <script src="/js/jquery.cookie.js"></script>
+  <link href="https://stackpath.bootstrapcdn.com/bootstrap/2.3.2/css/bootstrap.min.css" rel="stylesheet">
```

Comment:
A couple of note (from a noobie, feel free to correct my statements):
 - the official URI for v2.3.2 is different (but there is a redirection in place), see https://www.bootstrapcdn.com/legacy/bootstrap/
 - we could use SRI hashing (the default copying the official URI)
```
<link href="https://stackpath.bootstrapcdn.com/twitter-bootstrap/2.3.2/css/bootstrap-combined.min.css" rel="stylesheet" integrity="sha384-4FeI0trTH/PCsLWrGCD1mScoFu9Jf2NdknFdFoJhXZFwsvzZ3Bo5sAh7+zL8Xgnd" crossorigin="anonymous">
```

Reply:
thanks! i've switched to using this url. i agree using the official url + SRI hashes is better